hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -35,6 +37,7 @@
 _DB_NAME = 'data_v2.db'
 _CF_NAME_TX = b'tx'
 _CF_NAME_META = b'meta'
+_CF_NAME_STATIC_META = b'static-meta'
```

Comment:
We  need a migration to introduce this. The migration could simply just raise an exception saying it needs a sync from scratch. Alternatively we could make it so the storage serializes all metadata together like before with no changes to the format.

Reply:
There's a migration in the next PR: https://github.com/HathorNetwork/hathor-core/pull/1014

Since those new structures are not used here, this PR doesn't break the code.